plugin.rb

```diff
@@ -521,8 +521,15 @@ class ::ListController
     assigned_to_id = topic.assignment&.assigned_to_id
     assigned_to_type = topic.assignment&.assigned_to_type
 
-    if info[:user]&.id == assigned_to_id && assigned_to_type == "User"
-      TopicTrackingState.publish_assigned_private_message(topic, assigned_to_id)
+    assigned_to_ids =
+      if info[:user]&.id == assigned_to_id && assigned_to_type == "User"
+        [assigned_to_id]
+      elsif info[:group]&.id == assigned_to_id && assigned_to_type == "Group"
+        Group.find(assigned_to_id).users.pluck(:id)
```

Comment:
Should we maybe have a check to make sure the group is not deleted? Maybe it's not necessary if `info[:group]` is an actual Group record

Reply:
This is going to take two queries to execute. Can we get the user ids from the `GroupUser` table instead?

Reply:
I think we need to redo publishing for groups. If a group has 1000 users, this is going to send 1000 messages. Instead, it will be better if we can just publish the a group channel.